README.md

```diff
@@ -1,5 +1,5 @@
-# minicraft-plus-revived
-Minicraft+, a modded version of minicraft that adds many more features to the original version. To learn more about minicraft, or play it yourself, take a look at [www.playminicraft.com](https://www.playminicraft.com).
+# Minicraft Plus
+Minicraft+ is a modded version of Minicraft that adds many more features to the original version. To learn more about Minicraft, or play it yourself, take a look at all the [releases](https://github.com/chrisj42/minicraft-plus-revived/releases).
```

Comment:
I kinda feel bad not referencing the website, also, it's there because the site does in fact provide a lot of history on the game, not gone into here. Also, the download button links straight back to this releases page anyway.